core/network/rutracker/src/main/kotlin/flow/network/impl/RuTrackerInnerApiImpl.kt

```diff
@@ -198,5 +207,36 @@ internal class RuTrackerInnerApiImpl(private val httpClient: HttpClient) : RuTra
         const val CookieHeader: String = "Cookie"
     }
 
-    private fun String.toCp1251(): String = URLEncoder.encode(this, "Windows-1251")
+    private suspend inline fun HttpClient.submitForm(
```

Comment:
Fixed #18 by using right encoding for POST requests with `application/x-www-form-urlencoded`